vignettes/dp_merge.Rmd

```diff
@@ -102,23 +99,32 @@ prior to build the `dp_xcars` logic.
 ```{r dpbuild, eval = F}
 dp_connect(board_params = config$board_params, creds = config$creds)
 
+# specify which versions of data products to use
+version1 = "<version1>"
+version2 = "<version2>"
 dp_cars <- dp_get(board_params = config$board_params,
-                  data_name = "dp-cars-us001")
+                  data_name = "dp-cars-us001", version = version1)
```

Comment:
Note: caching the data before converting it to link is not necessary.

Reply:
Would you prefer we take out lines 105-109 and just reference the dp_link instead?

Reply:
We can take it out now or later as it is benign just unnecessary

Reply:
ok, I'll leave in for now